### ISSUE TITLE
Consider sending the rules/instructions when the first post is voted upon

### ISSUE THREAD

**Issue by Danyc0** [Danyc0/CSS-voting-bot#31]
Consider sending the rules/instructions when the first post is voted upon, as there might be a large time gap between registering (when they are currently sent), and the election. It's also not hugely clear if the info sent on registering is only for those planning to stand for posts

**Comment by antikitten** [comment]
I don't fully understand what this means, please elaborate for me.
Is it just posting a message that says "registering means you're just registering to vote not run for a position"?

**Comment by Danyc0** [comment]
Not really, it's basically just to consider if the bot should post the rules of voting just before the first ballot paper is sent (not all, because spam)
